tests/unit/components/sl-tab-panel-test.js

```diff
@@ -1,26 +1,16 @@
 import Ember from 'ember';
 import { moduleForComponent, test } from 'ember-qunit';
-import startApp from '../../helpers/start-app';
-import { contains } from '../../helpers/sl/synchronous';
-
-var App;
 
 moduleForComponent( 'sl-tab-panel', 'Unit - component: sl-tab-panel', {
-    needs: [ 'component:sl-tab-pane' ],
-
-    beforeEach: function() {
-        App = startApp();
-    },
-
-    afterEach: function() {
-        Ember.run( App, App.destroy );
-    }
+    needs: [ 'component:sl-tab-pane', 'template:components/sl-tab-pane' ]
 });
 
 test( 'Expected default classes are applied', function( assert ) {
-    var $component = this.render();
+    var component  = this.subject(),
+        $component = this.render();
 
-    assert.ok( contains( $component.prop( 'class' ), [ 'sl-tab-panel', 'sl-align-tabs-left' ] ), 'Default classes are not correctly applied' );
+    assert.ok( $component.hasClass( 'sl-tab-panel' ), 'Has class "sl-tab-panel"' );
+    assert.ok( $component.hasClass( 'sl-align-tabs-left' ), 'Has class "sl-align-tabs-left"' );
```

Comment:
This seemed more direct to me, and means the test doesn't rely on an extra function.